src/optimizer/OptRule.h

```diff
@@ -45,8 +45,8 @@ class OptRule {
 
 class RuleSet final {
 public:
-    static RuleSet &defaultRules();
-    static RuleSet &queryRules();
+    static RuleSet &DefaultRules();
+    static RuleSet &QueryRules();
```

Comment:
Emm, it seems not our naming style.

Reply:
this is the singleton function same as `Status::Error()`